app/src/main/java/comcmput301f16t01/github/carrier/Requests/RequestController.java

```diff
@@ -276,49 +271,38 @@ public void performAsyncUpdate() {
         gort.execute( UserController.getLoggedInUser().getUsername());
     }
 
-    /* * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * *
-     * * * * * * * * * * * * * * * * *    DEPRECATED FUNCTIONS   * * * * * * * * * * * * * * * * * *
-     * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * */
-
     /**
-     * Deprecated: use the void getSearchByLocation
+     * Ensures that the request controller on has results for requests within the price range of
+     * minPrice to maxPrice. (This affects getResult).
+     * @param minPrice The minimum value you wish to prune by.
+     * @param maxPrice The maximum value you wish to prune by. (Passing null is equivalent to passing positive infinity)
      */
-    @Deprecated
-    public ArrayList<Request> getSearchByLocation(Location location) {
-        throw new IllegalArgumentException( "This method is deprecated." );
-    }
-
-    /**
-     * Deprecated: There are several other functions that do this. Also, try to only use
-     * getSearchByKeyword or getSearchByLocation
-     */
-    @Deprecated
-    public ArrayList<Request> getAvailableRequests() {
-        throw new IllegalArgumentException( "This method is deprecated." );
-    }
-
-    /**
-     * Deprecated: Use getResult instead.
-     */
-    @Deprecated
-    public static RequestList getInstance() {
-        throw new IllegalArgumentException( "This method is deprecated." );
-    }
-
-    /**
-     * Deprecated: There is no user story that says we need to modify a request description after it has been
-     * created
-     */
-    @Deprecated
-    public void setRequestDescription(Request request, String description) {
-        throw new IllegalArgumentException( "This method is deprecated." );
+    public static void pruneByPrice(@NonNull Float minPrice, @Nullable Float maxPrice) {
+        RequestList filteredRequests = new RequestList();
```

Comment:
They are. You can see it here: #166.
After it is merged with master you wouldn't see these as green/red anymore.